packages/actor-rdf-parse-html-script/test/ActorRdfParseHtmlScript-test.ts

```diff
@@ -0,0 +1,211 @@
+        parseAction.input = stringToStream(`<script type="text/plain">{
+            "@id": "http://example.org/f",
+            "http://example.org/g": "http://example.org/h",
+            "http://example.org/i": "http://example.org/j"
+          }</script>`);
+
+        return actor.run(
+          { context: parseAction.context, handle: { input: parseAction.input }, handleMediaType: 'text/html' })
+          .then(async (output) => expect(await arrayifyStream(output.handle.quads)).toEqualRdfQuadArray([]));
+      });
+
+      it('should run with two JSON-LD scripts', () => {
+        parseAction.input = stringToStream(`<script type="application/ld+json">
+          [{
+              "@id": "http://example.org/a",
+              "http://example.org/b": "http://example.org/c",
+              "http://example.org/d": "http://example.org/e"
+          }]
+          </script>
+          <script type="application/ld+json">
+          [{
+              "@id": "http://example.org/f",
+              "http://example.org/g": "http://example.org/h",
+              "http://example.org/i": "http://example.org/j"
+          }]
+          </script>`);
+
+        return actor.run(
+          { context: parseAction.context, handle: { input: parseAction.input }, handleMediaType: 'text/html' })
+          .then(async (output) => expect(await arrayifyStream(output.handle.quads)).toEqualRdfQuadArray([
+            quad('http://example.org/a', 'http://example.org/b', 'http://example.org/c'),
+            quad('http://example.org/a', 'http://example.org/d', 'http://example.org/e'),
+            quad('http://example.org/a', 'http://example.org/b', 'http://example.org/c'),
+            quad('http://example.org/a', 'http://example.org/d', 'http://example.org/e'),
+          ]));
+      });
+
+      it('should run with one JSON-LD script and one text/plain script', () => {
+        parseAction.input = stringToStream(`<script type="application/ld+json">
+          [{
+              "@id": "http://example.org/a",
+              "http://example.org/b": "http://example.org/c",
+              "http://example.org/d": "http://example.org/e"
+          }]
+          </script>
+          <script type="text/plain">
+          [{
+              "@id": "http://example.org/f",
+              "http://example.org/g": "http://example.org/h",
+              "http://example.org/i": "http://example.org/j"
+          }]
+          </script>`);
+
+        return actor.run(
+          { context: parseAction.context, handle: { input: parseAction.input }, handleMediaType: 'text/html' })
+          .then(async (output) => expect(await arrayifyStream(output.handle.quads)).toEqualRdfQuadArray([
+            quad('http://example.org/a', 'http://example.org/b', 'http://example.org/c'),
+            quad('http://example.org/a', 'http://example.org/d', 'http://example.org/e'),
+          ]));
+      });
+
+      it('should run with two JSON-LD scripts and one N-Quads script', () => {
```

Comment:
The expected output of this test is wrong, right? The JSON-LD output is duplicated, and the N-Quads output is missing.